.github/workflows/Dockerfile.ci.alpine

```diff
@@ -46,10 +46,13 @@ RUN cp target/lldap /lldap/ && \
           web/static \
           /lldap/app/
 
+WORKDIR /lldap
 RUN set -x \
     && for file in $(cat /lldap/app/static/libraries.txt); do wget -P app/static "$file"; done \
     && for file in $(cat /lldap/app/static/fonts/fonts.txt); do wget -P app/static/fonts "$file"; done \
     && chmod a+r -R .
+RUN ls -al app/static && \
```

Comment:
What's the purpose of this `ls` ?

Reply:
it just to make sure, the file downloaded correctly.

Reply:
Then merge it with the `RUN` before (one fewer layer in the image).